workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/transform.py

```diff
@@ -297,3 +298,51 @@ def subset_variables(variables: Sequence, arg: DiagArg) -> DiagArg:
     prognostic_vars = [var for var in variables if var in prognostic]
     verification_vars = [var for var in variables if var in verification]
     return DiagArg(prognostic[prognostic_vars], verification[verification_vars], grid)
+
+
+def _is_3d(da: xr.DataArray):
+    return VERTICAL_DIM in da.dims
+
+
+@add_to_input_transform_fns
```

Comment:
Is this decorator needed now that you refer to this functions by symbol?

Reply:
The transform tests still use the registry to ensure any new transform has to pass tests- if we're ok with losing that automatic testing and having new additions be manually added to the list to test, `TRANSFORM_FNS` could be dropped altogether. As it stands now with the functions getting referred to directly, someone could still add a new transform that gets used in the diags but omit the decorator and also dodge the test coverage.